gearcmd/worker.go

```diff
@@ -43,6 +44,11 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 		data["error_message"] = err.Error()
 		data["value"] = 0
 		data["success"] = false
+		if conf.RetryCount > 0 {
+			log.Println(kayvee.FormatLog("gearcmd", kayvee.Error, "RETRY", data))
```

Comment:
Yeah they could be.  I could add the error to the data info being printed, then just print one `RETRY` message?